src/MainRouter.tsx

```diff
@@ -42,21 +42,22 @@ const MainRouter = (): ReactElement => {
   }, []);
 
   const checkBee = async () => {
-    try {
-      await fetch(
-        process.env.BEE_API_URL + "bytes/" + process.env.HEALTH_CHECK_DATA_REF
-      );
-      if (!isBeeRunning) {
-        setBeeRunning(true);
-        console.log("Bee is running");
-      }
-    } catch (error) {
+    fetch(
```

Comment:
why the await statement is removed ?

Reply:
I don't know there were some merge conflict here and I might made a mistake when solving it.

Reply:
I put  back await.